bindings/java/java/src/main/java/org/ethereum/evmc/EvmcVm.java

```diff
@@ -114,11 +113,11 @@ native void execute(
    * <p>This allows the context to managed in one method
    */
   public synchronized ByteBuffer execute(
-      HostContext context, int rev, ByteBuffer msg, ByteBuffer code, int size) {
```

Comment:
Note: this is a breaking change, but the Java binding we have is not stable enough and more changes are coming.

Everything merged so far were non-breaking changes.